FEATURE_MATRIX.md

```diff
@@ -15,7 +15,12 @@
 |              | Graph Model      | 🟢 Stable | 🟢 Stable   | 🟢 Stable |||
 |              | Graph Canvas     | 🟢 Stable | 🟢 Stable   | 🟢 Stable ||| 
 |              | Landscape Canvas | 🟢 Stable | 🟢 Stable   | 🟢 Stable ||| 
-|              | Script Canvas    | 🟢 Stable | 🟠 Volatile | 🟢 Stable || [Doc](https://o3de.org/docs/user-guide/scripting/script-canvas/) |
+|              | Script Canvas    | 🟢 Stable | 🟠 Volatile | 🟢 Stable || [Doc]| Installer    ||||||| 
+| Installer    ||||||| 
+|              | Installer  | 🟢 Stable | 🟢 Stable   | 🟢 Stable |||
```

Comment:
I thought Installer was under release?

Reply:
Installer is sig-build, Project Manager is sig-content. Sig-release does not own features, save the actual releases themselves.

Reply:
Ok will keep that in mind :D

Reply:
I'll remove it